src/client/pages/AssessmentSection/DataTable/Table/Table.tsx

```diff
@@ -31,21 +33,27 @@ const Table: React.FC<Props> = (props) => {
   const showODP = useShowOriginalDatapoints()
 
   const countryIso = useCountryIso()
+
   const { print } = useIsPrint()
   const tableRef = useRef<HTMLTableElement>(null)
 
   const { headers, table } = parseTable({ countryIso, cycle, data, showODP, table: tableProps })
   const { secondary, name } = table.props
-  const displayTableExportButton = !secondary && !print && tableRef.current != null
+  const displayButtons = !secondary && !print && tableRef.current != null
```

Comment:
why not secondary tables ?

Reply:
it's not a good idea to use refs to decide wether to render a component.
when ref value changes, components do not re render

Reply:
> why not secondary tables ?

This was added previously, what should be the condition in regard of secondary tables?

Reply:
So it seems it was disabled for eg.
![image](https://user-images.githubusercontent.com/5508251/228566064-2b357663-d23a-4771-800e-e458dc3bff1e.png)

But I will enable it then